### PR TITLE
Add a table view (with tag frequency in parentheses) to the showtags page

### PR DESCRIPTION
**Pull request by alice-blue**
This is definitely not ready to merge. The intention is to add an option for viewing all the tags in a table, with the frequency of each tag in parentheses. The third column is pushed off to the right, I think because of the length of some of the tags which are actually URLs, but I'm not sure how to fix it.

Fixes https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/499

### CODE REVIEW

File: www/showtags [hunk 1]
```diff
@@ -119,19 +119,20 @@ if (count($tags) == 0) {
...
+        . "Click on a tag to list the games associated with the tag."
         . "<p><div class=indented><center>";
 
     global $nonce;
     echo "<style nonce='$nonce'>\n"
         . ".showtags__tag { margin: 0 1em 0 1em; white-space: nowrap; }\n"
         . "</style>\n";
 
-
+
```
Comment (salty-horse) [inline www/showtags:131]: please remove whitespace from the line

File: www/showtags [hunk 2]
```diff
@@ -140,21 +141,28 @@ if (count($tags) == 0) {
...
+            . "<font size=2>"
+            . "$td</a> ($freq)</font></a></td>";
+        if ($currentcolumn == $totalcolumns) {
+                echo "</tr>";
+            }
     }
-
 }
-
+echo "</table>";
```
Comment (salty-horse) [inline www/showtags:160]: table should be closed inside the `else` block, one line up, since that's where it was opened.

File: www/showtags [hunk 3]
```diff
@@ -140,21 +141,28 @@ if (count($tags) == 0) {
...
+        if ($currentcolumn == 1) {
+            echo "<tr>";
+        }
+        echo "<td>"
+            . "<a href=\"search?searchfor=tag:$tu\">"
+            . "<font size=2>"
+            . "$td</a> ($freq)</font></a></td>";
+        if ($currentcolumn == $totalcolumns) {
+                echo "</tr>";
+            }
```
Comment (salty-horse) [inline www/showtags:157]: please move indent back for the `echo </tr>` and the `}`

File: www/showtags [hunk 4]
```diff
@@ -140,21 +141,28 @@ if (count($tags) == 0) {
...
+        if ($nextcolumn > $totalcolumns) {
+            $nextcolumn = 1;
+        }
+        if ($currentcolumn == 1) {
+            echo "<tr>";
+        }
+        echo "<td>"
+            . "<a href=\"search?searchfor=tag:$tu\">"
+            . "<font size=2>"
+            . "$td</a> ($freq)</font></a></td>";
```
Comment (salty-horse) [inline www/showtags:154]: is `</a>` supposed to be closed twice here...?

Comment (alice-blue) [reply www/showtags]: Ok, I think I made all the changes you mentioned. Thank you.

**Comment by dfabulich** [comment]
I filed a PR targeting your PR https://github.com/alice-blue/ifdb/pull/1 … if you approve of my changes and merge them, I can merge all of the changes at once and deploy them to production.

**Comment by alice-blue** [comment]
@dfabulich If I have suggestions of other changes to make, what should I do? Merge first and then talk?

**Comment by dfabulich** [comment]
I've merged this along with my changes from the PR.

**Pull request merged**